src/commands/datacenters.test.js

```diff
@@ -0,0 +1,80 @@
+const should = require(`should`);
+const sinon = require(`sinon`);
+require(`should-sinon`);
+
+const util = require(`./../util`);
+const testSubject = require(`./datacenters`);
+
+// Scope vars that we’ll redefine on each test.
+let options, datacentersStub, Fastly;
+
+// Begin testing.
+describe(`datacenters`, () => {
+  // Before and after each test, we reset some stuff
+  // to ensure a fresh set of arguments. See .beforeEach()
+  // and .afterEach() near the bottom of this file.
+
+  it(`should stop without an API key`, (done) => {
+    delete options.apikey;
+    testSubject(options, Fastly, util);
+    util.apiKeyPresent.should.be.calledOnce();
+    Fastly.should.not.be.called();
+    done();
+  });
+
+  it(`should not stop without a Service ID`, (done) => {
+    delete options.serviceid;
+    testSubject(options, Fastly, util);
+    util.serviceIdPresent.should.not.be.called();
+    Fastly.should.be.calledOnce();
+    done();
+  });
+
+  it(`should instantiate Fastly with API key`, (done) => {
+    testSubject(options, Fastly, util);
+    Fastly.should.be.calledOnce();
+    Fastly.getCalls()[0].args.should.deepEqual([options.apikey]);
+    done();
+  });
+
+  it(`should instantiate ResponseHandler`, (done) => {
+    testSubject(options, Fastly, util);
+    util.ResponseHandler.should.be.calledOnce();
+    done();
+  });
+
+  it(`should invoke .datacenters()`, (done) => {
+    testSubject(options, Fastly, util);
+    datacentersStub.should.be.calledOnce();
+    Fastly.getCalls()[0].args.should.deepEqual([options.apikey]);
```

Comment:
We already assert that Fastly gets called with the API key as its only argument on line 36. We should not assert it again here. Let's remove this line.